### ISSUE TITLE
Only show needed fields when addding new storage backend

### ISSUE THREAD

**Issue by aarondoet** [TandoorRecipes/recipes#283]
**Is your feature request related to a problem? Please describe.**
Currently on `/new/storage` inputs for password, token and url are displayed, no matter if you have dropbox or nextcloud selected as method. Therefore a note is added to tell the user to just ignore it.

**Describe the solution you'd like**
When selecting dropbox only the dropbox related inputs should show, when selecting nextcloud only nextcloud related inputs should show.

**Comment by vabene1111** [comment]
yes that would be nice. Currently the form is a standard django form without any javascript magic but that would be something worth changing i guess.

**Comment by MarcusWolschon** [comment]
Having 2 "path" fields with 1 of them ignored and no instructions also makes it extremely confusing to set up.

**Comment by vabene1111** [comment]
yes, the external recipes feature has not seen much love for a long time, it works in its very basic way when setup correctly but at some point i will need to work on it again. But for now more people benefit from work on the internal recipe features so this is not a priority.

**Comment by vabene1111** [comment]
Issue closed, will be addressed in #2976
